model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -831,6 +831,36 @@ def deepspeed_result_to_tokens(result: Dict[str, Any]) -> List[TokenOutput]:
     return tokens
 
 
+def validate_completion_params(
```

Comment:
can you rename it to validate_and_update_completion_params since this is not a readonly function?